test/Microsoft.ML.Benchmarks/Microsoft.ML.Benchmarks.csproj

```diff
@@ -0,0 +1,18 @@
+﻿<Project Sdk="Microsoft.NET.Sdk" ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <LangVersion>7.2</LangVersion>
+    <TargetFramework>net472</TargetFramework>
```

Comment:
It's an exe, so it cannot be Netstandard. It could be .NET Core app, but NetFx app makes it easier to run it (as the project output is an exe. 
Also, the output of this project does not really matter: BenchmarkDotNet generates the actual program that is being run during the test, and the generated project can be .NET Core (it's just a change in the config I can make if we prefer .net core test).

Reply:
Still, the dependency of .NET 4.7 makes this code unrunable on Linux, right? Can't we make it dual-target? `netcoreapp` and `net472`?

Reply:
Makes sense. I will change the host app to .NET Core too.